koji/playbooks/make-modules-metadata.yaml

```
@@ -0,0 +1,24 @@
+---
+- hosts: localhost
+  gather_facts: no
+  tasks:
+    - name: Collect RPM names
+      command: "rpm --query --package {{ repository_path }}/*rpm --queryformat=\"%{name}-%{epochnum}:%{version}-%{release}.%{arch}\n\""
```

I don't understand why this works. `command` doesn't use a shell, so the `*.rpm` should never be resolved.

```suggestion
      command: "rpm --query --package {{ repository_path }}/*rpm --queryformat='%{name}-%{epochnum}:%{version}-%{release}.%{arch}\n'"
```

---

Perhaps rpm itself implements globbing?

---

Yes it does, and that scares me.